src/core/Notification/Notification/Notification.md

```diff
@@ -0,0 +1,97 @@
+```js
+import { Notification, Button } from 'suomifi-ui-components';
+
+<>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    buttonContent={<Button variant="secondary">Button</Button>}
+  >
+    <p>
+      The service will be temporarily unavailable on 5.6.2021 at 21.00
+      – 23.59 due to maintenance.
+    </p>
+  </Notification>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    buttonContent={
+      <>
+        <Button>Button</Button>
+        <Button variant="secondary">Button</Button>
+      </>
+    }
+  >
+    <p>
+      The service will be temporarily unavailable on 5.6.2021 at 21.00
+      – 23.59 due to maintenance.
+    </p>
+  </Notification>
+  <Notification
+    closeText="Close"
+    labelText="Maintenance notification"
+    buttonContent={<Button variant="secondary">Button</Button>}
+  >
+    <p>
```

Comment:
What are your thoughts behind using the paragraph inside the main content in some examples?

Reply:
Added Paragraph to every example

Reply:
I'm not sure why we should have `p` or `Paragraph` used in examples? 
Could someone shed some light here for reasoning as it seems not to be obvious when doing the PR?

Reply:
I think semantically its paragraph specially if you have multiple "paragraphs".